tasks/Functions-admin/manageAllowlist.js

```diff
@@ -10,27 +11,39 @@ async function addOrRemove(action, taskArgs) {
     throw Error("This command cannot be used on a local development chain.  Specify a valid network.")
   }
 
-  let overrides = undefined
+  const overrides = {
+    gasLimit: 1500000,
+  }
+
   if (network.name === "goerli") {
-    overrides = {
-      // be careful, this may drain your balance quickly
-      maxPriorityFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-      maxFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-    }
+    overrides.maxPriorityFeePerGas = ethers.utils.parseUnits("50", "gwei")
```

Comment:
If we decided to deprecate Goerli, let's just throw an Error for Goerli.

Reply:
This has to be updated across the entire repo and should be in a separate PR.  It should be included in Justin's PR to transition from Goerli to Sepolia.